library/psa_crypto_rsa.c

```diff
@@ -258,6 +259,66 @@ static psa_status_t rsa_export_public_key(
 #endif /* defined(BUILTIN_KEY_TYPE_RSA_KEY_PAIR) ||
         * defined(BUILTIN_KEY_TYPE_RSA_PUBLIC_KEY) */
 
+#if defined(BUILTIN_KEY_TYPE_RSA_KEY_PAIR)
+static psa_status_t psa_rsa_read_exponent( const uint8_t *domain_parameters,
+                                           size_t domain_parameters_size,
+                                           int *exponent )
+{
+    size_t i;
+    uint32_t acc = 0;
+
+    if( domain_parameters_size == 0 )
+    {
+        *exponent = 65537;
+        return( PSA_SUCCESS );
+    }
+
+    /* Mbed TLS encodes the public exponent as an int. For simplicity, only
```

Comment:
"values that fit in a 32-bit integer, which is larger than int on just about every platform" - more accurate to say "... is at least as large as int ..."? Bit of a nit-pick but on first reading this confused me.

Reply:
I wrote this sentence originally, it's just been moved here. `int` only fits 31-bit positive integers. The comment should have read something like “larger than what fits in int”.

Reply:
I will think about this a bit more and may address it in a follow-up PR.